### ISSUE TITLE
Discussion on scheduling

### ISSUE THREAD

**Issue by dydx** [RichardLitt/remote-workers-toolbox#1]
> Schedule. Block out time for tasks, and in particular for work. Don't always be on - make sure you schedule time to be off. The easiest way to fail to do this is not to schedule time at all.

I've worked in the past for a small agency (I now do contract web work on my own). I had a pretty harsh time with the agency about their scheduling-- I was "on call" from 7am until 4am. There was no guarantee that there was even anything for me to do during those hours, but by god, I needed to be available during that time (without any compensation, either).

That lasted about two months, wherein I might have made enough to pay a month's rent in rural Georgia (i.e. hardly anything).

In my day to day work, now, I find the scheduling is difficult to get right, often in the time between jobs.

Something I do to keep myself in the same "work-mode" between jobs is to poke around GitHub and look for things to work on / fix, or hash out ideas of my own for side projects.

I find it keeps me from slacking off too much between paid gigs, helps me get my name out there a bit more, as well as sharpens random skills.

One pit-fall I have definitely had with personal work vs paid work is that 'turn off' time-- with paid work, the compulsion to stay tuned in after-hours isn't as bad as it is when I'm doing personal work. I'll often find myself catching a second wind after dinner and trying to pin down some ideas or fix a personal project... often into the morning.

I suppose I really should just schedule some non-negotiable time to be turned off every day. No matter what, from Xpm until Yam, no code.

**Comment by RichardLitt** [comment]
I agree with your final statement. Trying to either do paid work or personal work all of the time is the fastest way to lead to burnout. I don't code between 4:30-6pm every day if I can help it - I can't think of a single time I got really good work during those hours. So I do something else. It works for me.
